packages/atoms/src/Modal/Modal.tsx

```diff
@@ -8,9 +8,9 @@ import { Box } from '../Box';
 type ModalProps = {
     children: ReactNode;
     isModalVisible: boolean;
```

Comment:
I would rename it to **isVisible** to have the same property name as in SlideDownModal.

Reply:
Good point. I missed this one.

Reply:
[78b4874](https://github.com/trezor/trezor-suite/pull/5493/commits/78b48746b378e2de684f639fee2ca68aea0f0780)